app/models/book.rb

```diff
@@ -2,4 +2,10 @@ class Book < ApplicationRecord
   validates :genere, :author, :image, :title, :publisher, :year, presence: true
 
   has_many :rents, dependent: :delete_all
+
+  def actual_rent
+    rents.where(book_id: id)
+         .find_by!('rents.from <= :current_date AND rents.to >= :current_date',
```

Comment:
Dont use the bang `!` here. A book could exist without an actual rent.
Using the bang version of `find_by`, if no record is found this will raise an error.